en/modules/ROOT/nav.adoc

```diff
@@ -95,3 +95,5 @@
 
 * xref:en:whitepaper:index.adoc[Whitepaper]
 ** xref:en:whitepaper:decidim-a-brief-overview.adoc[Decidim: a brief overview]
+
+* xref:en:releases:index.adoc[_Release Notes_]
```

Comment:
Any reason for doing this with italic?

Reply:
Not really. I'd like a visual hint to differentiate the "Guide" part of the docu and the "Reference" part (e.g. Release Notes, FAQ, Glossary, etc, that I'd keep at the end of the menu). Italics was the simplest solution, but not necessary or an optimal solution.

I can remove it.

Reply:
Ah ok, at the moment as this is only "reference" section we'll keep it without italic, in the future when we have other sections we could add it.